src/cls/IPM/Storage/ModuleTemplate.cls

```diff
@@ -194,33 +194,43 @@ Method AddWebApps(
 
     new $namespace
     set $namespace = "%SYS"
+    set (tMatchRoles,tDispatchClass,tAutheEnabled,tPath)=""
     for i=1:1:$listlength(tAppList) {
-        set app = ##class(Security.Applications).Get($listget(tAppList,i),.props)
-        set tAppName = props("Name")
+        set app = ##class(Security.Applications).Get($listget(tAppList,i), .props)
+        if $$$ISOK(app) {
+            set tAppName = props("Name")
+            set tMatchRoles = props("MatchRoles")
+            set tAutheEnabled = props("AutheEnabled")
+            set tDispatchClass = props("DispatchClass")
+            set tPath = props("Path")
+        }else{
+            set tAppName = $listget(tAppList,i)
+            set tAutheEnabled = 64 //Unauthenticated
```

Comment:
If we must have a default, I'd prefer 32 (password) over 64 (unauthenticated) - also consider using macros from %sySecurity.inc